src/library_glut.js

```diff
@@ -361,15 +367,11 @@ var LibraryGLUT = {
     window.addEventListener("mousedown", GLUT.onMouseButtonDown, true);
     window.addEventListener("mouseup", GLUT.onMouseButtonUp, true);
     // IE9, Chrome, Safari, Opera
-    window.addEventListener("mousewheel", GLUT.onMouseWheel, true);
+    window.addEventListener("mousewheel", GLUT.onMouseWheel, { passive: false });
     // Firefox
-    window.addEventListener("DOMMouseScroll", GLUT.onMouseWheel, true);
+    window.addEventListener("DOMMouseScroll", GLUT.onMouseWheel, { passive: false });
 
-    Browser.resizeListeners.push(function(width, height) {
-      if (GLUT.reshapeFunc) {
-        {{{ makeDynCall('vii') }}}(GLUT.reshapeFunc, width, height);
-      }
-    });
+    window.addEventListener("resize", GLUT.onResize, true);
```

Comment:
this listens for resizes on the window, but I think we may also want to get resizes on the canvas - which is the "window", from the app's perspective? IIRC that's what Browser.resizeListeners does?

Reply:
Yup, i have not thought about this case, so i re-added the eventlistener.

Reply:
I see, thanks!